esp-hal/src/uart.rs

```diff
@@ -421,6 +338,15 @@ where
         }
     }
 
+    /// Configure RTS pin
+    pub fn with_rts<RTS: OutputPin>(self, rts: impl Peripheral<P = RTS> + 'd) -> Self {
+        crate::into_ref!(rts);
+        rts.set_to_push_pull_output(crate::private::Internal);
```

Comment:
Please just use `crate::private::Internal` rather than including the full path throughout this module.